packages/modules/localstack/src/localstack-container.ts

```diff
@@ -50,4 +54,30 @@ export class StartedLocalStackContainer extends AbstractStartedContainer {
   public getConnectionUri(): string {
     return `http://${this.getHost()}:${this.getPort().toString()}`;
   }
+
+  /**
+   * Provides a default region that is preconfigured to communicate with a given simulated service.
+   * @returns A default region
+   */
+  public getRegion(): string {
+    return this.startedTestContainer.getEnvironment()["DEFAULT_REGION"] || DEFAULT_REGION;
+  }
+
+  /**
+   * Provides a default access key that is preconfigured to communicate with a given simulated service.
+   * <a href="https://github.com/localstack/localstack/blob/master/doc/interaction/README.md?plain=1#L32">AWS Access Key</a>
```

Comment:
I'm getting a 404 for these URLs. I also couldn't find reference to these environment variables in the LS config

Reply:
Sorry, I blindly copy/paste here the comments coming from testcontainers-java/localstack